Sources/TensorFlow/Operators/Image.swift

```diff
@@ -0,0 +1,331 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public enum ResizeMethod {
+  case nearest, bilinear, bicubic, lanczos3, lanczos5, gaussian, mitchellcubic
+}
+
+/// Resize images to size using the specified method.
+///
+/// - Parameters:
+///   - images: 4-D Tensor of shape [batch, height, width, channels] or 3-D Tensor of shape [height, width, channels].
+///   - size: A tuple of two Int: (new_height, new_width). The new size for the images.
+///   - method: ResizeMethod. Defaults to bilinear.
```

Comment:
Use English phrases and sentences in doc comments.
Always use backticks around code.
The doc comment template for default parameter values is: ``The default value is `xxx`.``
The doc comment should show `.bilinear` instead of just `bilinear`.
```suggestion
///   - method: The resize method. The default value is `.bilinear`.
```